content/articles/blockchain-as-a-form-of-e-voting/index.md

```diff
@@ -0,0 +1,77 @@
+# Blockchain as a form of e-voting
+
+### **Introduction**
+
```

Comment:
No need to put asterics (**) on headings as they are bold by default. Please remove them in all your headings.